docs/toc.md

```diff
@@ -223,7 +223,7 @@
 ### [Documenting your code](csharp/codedoc.md)
 ### [Versioning](csharp/versioning.md)
 <!-- End of C# Concepts section -->
-<!--## [🔧 Using the .NET Compiler Platform](csharp/roslyn/index.md)-->
+<!--## [🔧 Using the .NET Compiler](csharp/roslyn/index.md)-->
```

Comment:
This is Roslyn; the official name of the technology is ".NET Compiler Platform".